src/app/modules/auth/login.page.tsx

```diff
@@ -1,3 +1,34 @@
+import { Button } from "../../../atomic/atm-button";
```

Comment:
https://medium.com/thefork/escaping-relative-import-paths-hell-in-javascript-9f258baeb15e

Dá uma olhadinha nesse link é bem legal sobre import relativo e absoluto